.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "react-app",
+	"ignorePatterns" :["dist/*"],
+	"env": {
+		"browser": true,
+		"node": true
+	},
+  "rules": {
+    "quotes": [1, "single"],
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1,
+    "react/react-in-jsx-scope": 1
```

Comment:
Why do we have this enabled? We use a recent enough version of react to be able to omit `import React from "react";` everywhere and vite seems to be able to handle just fine.

Reply:
The only reason I added this, is because it was present in the rc file before it was removed in #111. If we can omit it, I am certainly in favour of removing it.

Reply:
Do you propoes that all `react/` things should be removed?

Reply:
> Do you propose that all `react/` things should be removed?

GitHub did an oopsie and didn't show me this comment before. Anyhow, the middle one is needed though it is already in the recommended configuration for `eslint-plugin-react`, the other two can be disabled by extending the `jsx-runtime` configuration as described [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md#when-not-to-use-it).

Reply:
I will have a look into these links

Reply:
I hope I understood the extending correctly. Did my latest commit do what you expected?